src/main/scala/viper/gobra/ast/frontend/Ast.scala

```diff
@@ -691,6 +691,11 @@ case class PFunctionSpec(
                       isPure: Boolean = false,
                       ) extends PSpecification
 
+case class PBodyParameterInfo(
+                           /** stores indices of parameters that are used as addressable variables in the code body */
+                           addressedParameters: Vector[PIdnUse]
```

Comment:
Imho `addressableParameters` or `sharedParameters` would be better

Reply:
For me, this is the same as for `isParaemterShared`. The parameter itself is not shared.

Reply:
I will change it to `addressableParameters`

Reply:
Or how about `sharableParameters`?

Reply:
Then we at least stick to the same terminology. But add a comment that this means that these params have been declared as shared in the body (because sharable doesn‘t sound at all like some declaration things)

Reply:
Is the new name and description fine for you?

Reply:
Yes